Documentation/subcommands/run.md

```diff
@@ -80,11 +80,13 @@ To inherit all environment variables from the parent use the `--inherit-env` fla
 
 To explicitly set individual environment variables use the `--set-env` flag.
 
+To explicitly set environment variables from a file use the `--set-env-file` flag. Variables are expected to be in the OS format (e.g. VAR_NAME=VALUE) and lines starting with `#` or `;` will be ignored.
```

Comment:
Te problem is that (EnvironmentFile=)[https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=] is much more complex and flexible. It allows to use other "magic" characters, trails white spaces and other things that here are not supported.